docs/UserGuide.md

```diff
@@ -123,17 +126,18 @@ Examples:
 
 ### Deleting a patient or specialist : `delete`
 
-Deletes the specified patient or specialist from the stored records.
+Deletes the specified patients or specialists from the stored records.
 
-Format: `delete INDEX`
+Format: `delete INDEX…​`
 
-* Deletes the person at the specified `INDEX`.
+* Deletes all persons at the specified `INDEX…​`.
 * The index refers to the index number shown in the displayed person list.
 * The index **must be a positive integer** 1, 2, 3, …​ with a maximum value of the list size.
+* The indexes must **not** contain any duplicate integers.
 
 Examples:
-* `list -pa` followed by `delete 2` deletes the 2nd patient in the listed patients. 
-* `find -sp n/Betsy` followed by `delete 1` deletes the 1st specialist from the specialists listed in the `find` command.
+* `list -pa` followed by `delete 2` deletes the 2nd patient in the listed patients.
+* `find -sp s/s/Orthopaedic` followed by `delete 2 3 4` deletes the 2nd, 3rd and 4th specialist listed in the `find` command.
```

Comment:
Looks like a typo here

Reply:
Thank you for pointing that out! I have fixed the typo in the User Guide.